icons/cloudbees.svg

```diff
@@ -1 +1 @@
-<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>CloudBees icon</title><path d="M3.55 6.41c0-1.2.73-2.28 1.88-2.8A3.51 3.51 0 0 1 8.94 4l2.52-2.35A7.25 7.25 0 0 0 3.22.98a6.22 6.22 0 0 0-2.95 7.2 6.82 6.82 0 0 0 6.6 4.64h2.58v-3.3H6.87C5.04 9.5 3.55 8.11 3.55 6.4zm13.51 4.73h-2.58v3.3h2.58c1.83 0 3.32 1.4 3.32 3.1 0 1.72-1.49 3.1-3.32 3.1s-3.32-1.37-3.32-3.08V6.4c0-1.58-.63-3.11-1.76-4.29L9.46 4.48c.47.53.73 1.22.73 1.93v11.14c0 3.54 3.08 6.41 6.87 6.41 3.8 0 6.87-2.87 6.87-6.41s-3.07-6.41-6.87-6.41z"/></svg>
\ No newline at end of file
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>CloudBees icon</title><path d="M3.5615 6.4196c0-1.203.732-2.284 1.884-2.805a3.516 3.516 0 013.5161.39l2.5241-2.3541a7.2632 7.2632 0 00-8.2542-.67 6.2302 6.2302 0 00-2.955 7.2111 6.8322 6.8322 0 006.611 4.6481h2.5852v-3.305H6.8876c-1.833-.02-3.326-1.413-3.326-3.1251zm13.5343 4.738h-2.586v3.3052h2.585c1.833 0 3.3261 1.403 3.3261 3.106 0 1.723-1.493 3.1051-3.326 3.1051-1.8331 0-3.3261-1.372-3.3261-3.085V6.4085c0-1.582-.63-3.1151-1.763-4.2971l-2.5231 2.374c.47.531.731 1.222.731 1.934v11.1593c0 3.5461 3.085 6.4212 6.8821 6.4212 3.8071 0 6.8822-2.875 6.8822-6.4212 0-3.546-3.075-6.4211-6.8822-6.4211z"/></svg>
```

Comment:
Any chance you can regenerate from source? Looks like the logo has changed.
Path below has been generated at 3 decimals...
```svg
M13.759 10.221zM24 16.97c0 3.73-3.093 6.747-6.904 6.747-3.812 0-6.908-3.017-6.908-6.747V6.979a3.215 3.215 0 00-.531-1.75c-.225-.352-.258-.239.065-.554l1.922-1.882a.358.358 0 01.029-.026c.416-.342.299-.351.649.054.923 1.156 1.437 2.664 1.437 4.158v3.242h-.001.001v.025-.025h3.337c3.811 0 6.904 3.02 6.904 6.749zm-3.565 0c0-1.745-1.405-3.158-3.167-3.246v.011h-3.509v3.22c0 1.803 1.495 3.279 3.337 3.279 1.842-.001 3.339-1.462 3.339-3.264zM3.549 7.05c0-1.804 1.488-3.27 3.319-3.27.735 0 1.378.188 1.929.565l2.511-2.472C10.059.812 8.579.283 6.869.283 3.081.283 0 3.32 0 7.05c0 3.732 3.081 6.767 6.869 6.767h2.429v-3.496H6.869c-1.831 0-3.32-1.467-3.32-3.271z
```

Red is PR, black is built from source.
![image](https://user-images.githubusercontent.com/1830867/102828601-ffe50f80-43dc-11eb-8d41-b0ba6395dafa.png)